### ISSUE TITLE
"Error: /remote:new invalid alias format" when adding a new connection

### ISSUE THREAD

**Issue by bdmorin** [wavetermdev/waveterm#202]
**Describe the bug**
New User here.
When I create a remote connection, I fill in the forms and receive this error:
```
Error: /remote:new invalid alias format
```

**To Reproduce**
1. Go to connections
2. Add new connection
3. Fill in boxes

I used a private key, and a public key. Is ED not supported?

**Expected behavior**
A new connection is made using a private key to the host detailed in the setup.

**Screenshots**

![CleanShot 2023-12-28 at 15 34 56@2x](https://github.com/wavetermdev/waveterm/assets/476219/f7cde4bb-e430-46bc-809a-cdf997c55b9f)

**Desktop (please complete the following information):**
 - OS:      macOS 14.2.1 (23C71)
 - Version: Darwin 23.2.0
 - ssh:     OpenSSH_9.4p1, LibreSSL 3.3.6
 
**Additional context**
I don't have a host config in .ssh/config for 10.0.0.8 here is my global host config.

```
Host *
    controlmaster auto
    controlpath /Users/bdmorin/.ssh/.tmp/%C
    controlpersist yes
    forwardagent yes
    forwardx11 no
    IdentityAgent "~/.1password/agent.sock"
    serveraliveinterval 5
    UserKnownHostsFile /dev/null
    StrictHostKeyChecking no
    addkeystoagent yes
    sendenv LS_COLORS
```
I don't know if waveterm uses .ssh/config file.

**Comment by bdmorin** [comment]
Oh - it appears that once I made the alias a name instead of an IP address it worked.

**Comment by bdmorin** [comment]
Well, something new.

![CleanShot 2023-12-28 at 15 53 16@2x](https://github.com/wavetermdev/waveterm/assets/476219/4d98e8ba-b1e5-4210-ac95-49b897244c39)

It seems fish shell is incompatible with the install process?
```
prompt> console for connection [ph0x]
prompt> connecting to bdmorin@10.0.0.8...
Warning: Permanently added '10.0.0.8' (ED25519) to the list of known hosts.
prompt> error connecting to remote: invalid packet received from mshell client: raw[fish: Unsupported use of '='. In fish, please use 'set PATH $PATH:~/.mshell'.]
prompt> connecting to bdmorin@10.0.0.8...
prompt> error connecting to remote: invalid packet received from mshell client: raw[fish: Unsupported use of '='. In fish, please use 'set PATH $PATH:~/.mshell'.]
prompt> installing mshell v0.3.0 to bdmorin@10.0.0.8...
prompt> error, install failed: no response packet received from client
```

**Comment by bdmorin** [comment]
The fish shell issue is a dupe of https://github.com/wavetermdev/waveterm/issues/92
Sorry about that. 

I guess the final bug report is that if you use an IP address as an alias it throws an error.

**Comment by oneirocosm** [comment]
as a note, this will not be available in the next release (should be 0.6.0), but it can be found in the main github branch until then
